app/Shipments/Http/Requests/ShipmentRequest.php

```diff
@@ -69,4 +69,20 @@ public function token(): Token
 
         return $token;
     }
+
+    /**
+     * @return int|null
+     */
+    public function pageNumber(): ?int
+    {
+        return $this->query('pageNumber');
```

Comment:
the `query()` method here will return a string, therefore the function will break because of the type. We need to typecast to int before returning.
```suggestion
        return $this->query('pageNumber') ? (int) $this->query('pageNumber') : null;
```